web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -56,6 +62,8 @@ import {
   selectUserProfile
 } from '../../../context/selectors';
 
+import { titleCase } from '../../../helpers/strings.js';
```

Comment:
This is used to convert a string status value that is all uppercase to one that uses title case.